tests/serialization/test_predictors.py

```diff
@@ -49,39 +50,45 @@ def test_graph_serialization(valid_graph_predictor_data):
 
 def test_expression_serialization(valid_expression_predictor_data):
     """Ensure that a serialized ExpressionPredictor looks sane."""
-    predictor_serialization_check(valid_expression_predictor_data, ExpressionPredictor)
+    predictor_node_serialization_check(valid_expression_predictor_data, ExpressionPredictor)
 
 
 def test_ing_to_formulation_serialization(valid_ing_formulation_predictor_data):
     """Ensure that a serialized IngredientsToFormulationPredictor looks sane."""
-    predictor_serialization_check(valid_ing_formulation_predictor_data, IngredientsToFormulationPredictor)
+    predictor_node_serialization_check(valid_ing_formulation_predictor_data, IngredientsToFormulationPredictor)
 
 
 def test_mean_property_serialization(valid_mean_property_predictor_data):
     """Ensure that a serialized MeanPropertyPredictor looks sane."""
-    predictor_serialization_check(valid_mean_property_predictor_data, MeanPropertyPredictor)
+    predictor_node_serialization_check(valid_mean_property_predictor_data, MeanPropertyPredictor)
 
 
 def test_simple_mixture_predictor_serialization(valid_simple_mixture_predictor_data):
-    predictor_serialization_check(valid_simple_mixture_predictor_data, SimpleMixturePredictor)
+    predictor_node_serialization_check(valid_simple_mixture_predictor_data, SimpleMixturePredictor)
 
 
 def test_label_fractions_serialization(valid_label_fractions_predictor_data):
     """Ensure that a serialized LabelFractionPredictor looks sane."""
-    predictor_serialization_check(valid_label_fractions_predictor_data, LabelFractionsPredictor)
+    predictor_node_serialization_check(valid_label_fractions_predictor_data, LabelFractionsPredictor)
 
 
 def test_ingredient_fractions_serialization(valid_ingredient_fractions_predictor_data):
     """"Ensure that a serialized IngredientsFractionsPredictor looks sane."""
-    predictor_serialization_check(valid_ingredient_fractions_predictor_data, IngredientFractionsPredictor)
+    predictor_node_serialization_check(valid_ingredient_fractions_predictor_data, IngredientFractionsPredictor)
 
 
 def test_auto_ml_serialization(valid_auto_ml_predictor_data):
     """"Ensure that a serialized AutoMLPredictor looks sane."""
-    predictor_serialization_check(valid_auto_ml_predictor_data, AutoMLPredictor)
+    predictor_node_serialization_check(valid_auto_ml_predictor_data, AutoMLPredictor)
 
 
-def test_invalid_predictor_type(invalid_predictor_data):
+def test_invalid_graph(invalid_graph_predictor_data):
+    """Ensures we raise proper exception when horribly malformed data is used."""
```

Comment:
```suggestion
    """Ensures we raise proper exception when malformed data is used."""
```

I hope that it doesn't need to be horribly malformed in order to raise an exception :p